apps/condo/domains/billing/schema/AllResidentBillingReceiptsService.js

```diff
@@ -95,7 +95,7 @@ const AllResidentBillingReceiptsService = new GQLCustomSchema('AllResidentBillin
             resolver: async (parent, args, context = {}) => {
                 const { where, first, skip, sortBy } = args
 
-                const serviceConsumerWhere = get(where, 'serviceConsumer', {})
+                const serviceConsumerWhere = { ...get(where, 'serviceConsumer', {}), deletedAt: null }
```

Comment:
get

Reply:
it's back not front
If we plan to remove all usages of get - then it will be another task for this
in this PR I used old code with minimum corrections